docs/visualization/wcsaxes/overlays.rst

```diff
@@ -300,7 +300,7 @@ image to plot the contours for:
 Spherical patches
 *****************
 
-In the case where you are making a plot of a celestial image, and want to plot a circle that represents the area within a certain angle of a longitude/latitude, the `~matplotlib.patches.Circle` patch is not appropriate, since it will result in a distorted shape (because longitude is not the same as the angle on the sky). For this use case, you can instead use `~astropy.visualization.wcsaxes.SphericalCircle`, which takes a tuple of `~astropy.units.Quantity` as the input, and a `~astropy.units.Quantity` as the radius:
+In the case where you are making a plot of a celestial image, and want to plot a circle that represents the area within a certain angle of a longitude/latitude, the `~matplotlib.patches.Circle` patch is not appropriate, since it will result in a distorted shape (because longitude is not the same as the angle on the sky). For this use case, you can instead use `~astropy.visualization.wcsaxes.SphericalCircle`, which takes a tuple of `~astropy.units.Quantity` or a `~astropy.coordinates.SkyCoord` object as the input, and a `~astropy.units.Quantity` as the radius:
```

Comment:
The file https://github.com/astropy/astropy/blob/9819e926cb49fbc36019f5e641b563975e56d57c/docs/common_links.txt provides many useful aliases. It was added quite recently so a lot of the documentation does not make use of it yet, but I see no reason why new documentation shouldn't. Also this line looks very long, but if the rest of the file also contains overlong lines then it might be best to leave the line length as it is for consistency.

```suggestion
In the case where you are making a plot of a celestial image, and want to plot a circle that represents the area within a certain angle of a longitude/latitude, the `~matplotlib.patches.Circle` patch is not appropriate, since it will result in a distorted shape (because longitude is not the same as the angle on the sky). For this use case, you can instead use `~astropy.visualization.wcsaxes.SphericalCircle`, which takes a tuple of |Quantity| or a |SkyCoord| object as the input, and a |Quantity| as the radius:
```